collector/database/classes/collector.php

```diff
@@ -72,4 +89,33 @@ public function get_metrics($metricnames = null, $since = false): array {
                ORDER BY time asc";
         return $DB->get_records_sql($sql, $params);
     }
+
+    /**
+     * Records retrieved data in collector.
+     *
+     * @param  object $metricclass Class representing metric.
+     * @param  array $data Array of metric items.
+     */
+    public function record_saved_metrics(object $metricclass, array $metricitems = []) {
```

Comment:
'object' as a type hint is not supported in older versions of PHP. It should be '\tool_cloudmetrics\metric\base' instead.

Reply:
Fixed.